extensions/notebook/src/common/utils.ts

```diff
@@ -189,7 +190,8 @@ export function sortPackageVersions(versions: string[], ascending: boolean = tru
 }
 
 export function isEditorTitleFree(title: string): boolean {
-	let hasTextDoc = vscode.workspace.textDocuments.findIndex(doc => doc.isUntitled && doc.fileName === title) > -1;
+
+	let hasTextDoc = vscode.workspace.textDocuments.findIndex(doc => doc.isUntitled && doc.fileName === title && !notebookLanguages.find(l => l === doc.languageId)) > -1;
```

Comment:
Could we use "language" or "lang" in the find callback? Just using "l" looks like a 1 at a glance.